GenderPayGap.WebUI/wwwroot/styles/components/_guidance.scss

```diff
@@ -60,3 +60,28 @@
         }
     }
 }
+
+.print-link {
+    background: url(../../img/icon-print.png) no-repeat 0 50%;
```

Comment:
If there's any images we're using on new-style pages, it would be great to put them in the `wwwroot/assets/images` folder (so it's easier to see which are old and which are new)